src/collection.js

```diff
@@ -589,12 +589,18 @@ export default class Collection {
    *     Conflicting server records will be overriden with local changes.
    *   * `Collection.strategy.MANUAL`:
    *     Conflicts will be reported in a dedicated array.
+   * - {Boolean} forceBackoff: Force synchronization even if server is currently
+   *   backed off.
    *
    * @param  {Object} options Options.
    * @return {Promise}
    */
-  sync(options={strategy: Collection.strategy.MANUAL, headers: {}}) {
-    // TODO rename options.mode to options.strategy
+  sync(options={strategy: Collection.strategy.MANUAL, headers: {}, forceBackoff: false}) {
+    // Handle server backoff: XXX test
```

Comment:
That's an obsolete comment which should be removed. Maybe you can just explain me how to enable it on a local kinto instance so I can quickly test that everything works as expected.